src/reducers/SelectedRevisionsSlice.ts

```diff
@@ -23,26 +24,37 @@ const selectedRevisions = createSlice({
       state.base = [action.payload.selectedRevisions[0]];
       //returns array without first element
       state.new = action.payload.selectedRevisions.slice(1);
+      state.editModeRevisions = action.payload.selectedRevisions;
     },
-    deleteRevision(
+    saveUpdatedRevisions(
```

Comment:
I think it would help the reader if each action had a comment explaining when it's called (for example: "when the user press the save button when the search form is in edit mode").

Reply:
Good idea.